src/Microsoft.Azure.WebJobs.Host/IJobHost.cs

```diff
@@ -1,12 +1,16 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+
 namespace Microsoft.Azure.WebJobs
 {
-    /// <summary>
-    /// Marker type for Table resolution
-    /// </summary>
-    public sealed class ODataFilterResolutionPolicy
+    public interface IJobHost
```

Comment:
How does this differ from the IHost interface returned from IHostBuilder? Seems like the same interface the IJobHost interface is obsolete.

Reply:
They are different. `IJobHost` is a WebJobs interface and provides the public API surface relevant to WebJobs.